### ISSUE TITLE
Switch vignette to R Markdown and add more examples

### ISSUE THREAD

**Issue by nuest** [ropensci/suppdata#44]
The vignette is currently an `Rnw` file, which is fine, but also prevents it to be rendered nicely, e.g. as part of a `pkgdown`-Website (if we ever want one).

The actual advantage I would see is that an R Markdown vignette is much more accessible to newcomers. With some more examples, it would allow us to add a Badger for mybinder.org, which could open the vignette in RStudio and very nicely demonstrate the package. Try out the following link to demonstrate this (must navigate to vignette manually, but cannot compile PDF because there is no LaTeX):

https://mybinder.org/v2/gh/ropensci/suppdata/master?urlpath=/rstudio

There are more than enough examples in the tests that we could pick to show off in the vignette. An Rmd vignette can be run easily (by chunk) or completely (by installing knitr etc.)

The badge would be 

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ropensci/suppdata/master?filepath=vignettes%2Fsuppdata-intro.Rnw&urlpath=/rstudio)

**Comment by willpearse** [comment]
Sure, I would be fine with moving it over to Markdown. I'm not flagging this as a personal high priority, but I would be happy to do it. The easier to use the better, for sure.

**Comment by willpearse** [comment]
Does the launch of this new ROpenSci documentation integration site (https://ropensci.org/technotes/2019/06/07/ropensci-docs/) change your opinion on this? I think our auto-generated one is pretty awesome (https://docs.ropensci.org/suppdata/). No badge though :-(

**Comment by nuest** [comment]
Yeah, the new docs website is really cool. Agree with you that the switch to R Markdown is not a high priority.

**Comment by willpearse** [comment]
Great, thanks!

>
